.github/workflows/golang-testing.yml

```diff
@@ -39,17 +39,62 @@ jobs:
 
     - name: Run govulncheck
       run: |
-        govulncheck -test netflow/...
-        govulncheck -test manager/...
-        govulncheck -test ipexist/...
-        govulncheck -test generators/...
-        govulncheck -test filewatch/...
-        govulncheck -test client/...
-        govulncheck -test chancacher/...
-        govulncheck -test timegrinder/...
-        govulncheck -test ingest/...
-        govulncheck -test ingesters/...
-        GOOS=windows govulncheck -test winevent/...
+        govulncheck -test ./netflow/...
+        govulncheck -test ./manager/...
+        govulncheck -test ./ipexist/...
+        govulncheck -test ./generators/ipgen/...
+        govulncheck -test ./generators/base/...
+        govulncheck -test ./generators/gravwellGenerator/...
+        govulncheck -test ./filewatch/...
+        govulncheck -test ./client/...
+        govulncheck -test ./chancacher/...
+        govulncheck -test ./timegrinder/...
+        govulncheck -test ./ingest
+        govulncheck -test ./ingest/entry/...
+        govulncheck -test ./ingest/config/...
+        govulncheck -test ./ingest/log
+        govulncheck -test ./ingest/processors
+        govulncheck -test ./ingest/processors/tags
+        govulncheck -test ./ingest/processors/plugin
+        govulncheck -test ./tools/...
+        govulncheck -test ./kitctl/...
+        govulncheck -test ./migrate/...
+        govulncheck -test ./ingesters
```

Comment:
can't run it on a dir with no actual code

Reply:
good catch, fixed.